src/ex4/server/routes/api.js

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+const itemRouter = express.Router();
+import {
+  createItem,
+  deleteItem,
+  getAllItems,
+  deleteAllItems,
+  sortItems,
+} from "../controllers/itemController.js";
+
+itemRouter.post("/", createItem);
```

Comment:
I recommend use this syntax as it allows you to handle the response and more readble
<img width="374" alt="image" src="https://user-images.githubusercontent.com/73115367/174746070-493d29de-f944-48f3-8589-85c77a05a6d9.png">